### PR TITLE
Completed javascript-if. | Seungjoo Kim

### PR DESCRIPTION
**Pull request by SeungjooKimSJ**
image

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  } else if (pH > 7 && pH <= 14) {
+    return 'base';
+  } else if (pH < 0 || pH > 14) {
+    return 'invalid pH level';
+  }
+}
+// console.log(categorizeAcidity(-1));
+// console.log(categorizeAcidity(5));
+// console.log(categorizeAcidity(7));
+// console.log(categorizeAcidity(10));
```
Comment (thebearingedge) [inline javascript-if/main.js:93]: Please delete all commented out code.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person.age >= 21 && person.age >= 16) {
+    return false;
+  } else {
+    return false;
+  }
```
Comment (codingcodymiller) [inline javascript-if/main.js:55]: This one was a joke, you can just `return false` since that is always the end result

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
One more minor change!

File: javascript-if/main.js [hunk 3]
```diff
@@ -50,8 +50,6 @@ function isOldEnoughToDrive(person) {
 function isOldEnoughToDrinkAndDrive(person) {
   if (person.age >= 21 && person.age >= 16) {
     return false;
-  } else {
-    return false;
   }
```
Comment (codingcodymiller) [inline javascript-if/main.js]: The only code needed here is `return false`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Seungjoo! Still needs adjustment, please see my note

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**